<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchToolbar</class>
 <widget class="QWidget" name="SearchToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>40</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="MacToolButton" name="closeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="FocusSelectLineEdit" name="lineEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Find...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="next">
     <property name="toolTip">
      <string>Find the next match for the current search phrase</string>
     </property>
     <property name="text">
      <string>&amp;Next</string>
     </property>
     <property name="icon">
      <iconset theme="go-down-search"/>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="previous">
     <property name="toolTip">
      <string>Find the previous match for the current search phrase</string>
     </property>
     <property name="text">
      <string>&amp;Previous</string>
     </property>
     <property name="icon">
      <iconset theme="go-up-search"/>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="caseSensitive">
     <property name="text">
      <string>&amp;Match Case</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FocusSelectLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mc.tools.FocusSelectLineEdit</header>
  </customwidget>
  <customwidget>
   <class>MacToolButton</class>
   <extends>QToolButton</extends>
   <header>mc.tools.MacToolButton</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
